types/atom/index.d.ts

```diff
@@ -121,6 +121,9 @@ export interface AtomEnvironment {
     /** Returns a boolean that is true if the current window is running specs. */
     inSpecMode(): boolean;
 
+    /** Get the full name of this Atom release (e.g. "Atom", "Atom Beta") */
+    getAppName(): string;
```

Comment:
This function was first available in Atom version `v1.40.0`. To bump the definition version from `v1.36.0` to `v1.40.0`, you'll need to replace the three occurrences of `1.36` within the file with `1.40`.